src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -22,6 +23,7 @@ import {colors} from '../../utils/colors';
 import {getCommentLength, getCountCommentWithChild} from '../../utils/getstream';
 import {showScoreAlertDialog} from '../../utils/Utils';
 import {normalizeFontSizeByWidth} from '../../utils/fonts';
+import usePostHook from '../../hooks/core/post/usePostHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The custom hook `usePostHook` is imported here. This hook provides functions for handling follow/unfollow actions on posts and topics. It's a good practice to encapsulate such logic in hooks, which can be reused across different components.
```diff
+ import usePostHook from '../../hooks/core/post/usePostHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->